### ISSUE TITLE
Dunning use of currency is inconsistent.

### ISSUE THREAD

**Issue by mckayERP** [adempiere/adempiere#2362]
If the Currency is specified in the Create Dunning Run process, the invoices will be filtered by currency but the payments aren't.  Is this a problem?

**Comment by adaxa-steven** [comment]
yes! but the fix is not obvious.  if the statement has more than one currency you can not add the values and get a sensible total ... which pushes you to create a statement per currency .. but if you only ever invoice a customer in (say) USD and he pays in EUR and an EUR payment has not been allocated then you might want the payment to be shown.  By default I think it would be better to filter the payments to the selected currency.
